extractors/main.py

```diff
@@ -6,6 +6,7 @@ def main():
     cnac = ExtractCNAC()
     cnac.download()
 
+    # todo: where is this date() defined? is crashing the script whenever run using docker.
```

Comment:
`date` is a method from Python's built-in `datetime` module. It doesn't require any package installations. 
Should've simply been imported as `import datetime`, and the call should be `[...] datetime.date(yyyy, dd, M)`.
Let's merge this PR to get everyone on the same page with regards to the environments and runtimes, and then make a thorough review of the code.